salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc

```diff
@@ -0,0 +1,1164 @@
+** <<Close Job - Bulk API>> +
+Shows how to update the status of the specified job ID so that the job is closed and queued for processing.
+** <<Abort Job - Bulk API>> +
+Shows how to update the status of the specified job ID so that the job is aborted and not queued for processing.
+** <<Get Job - Bulk API>> +
+Shows how to retrieve current status of the specified job ID.
+** <<Delete Job - Bulk API>> +
+Shows how to delete or close the specified job ID, which deletes job data and metadata stored by Salesforce.
+* Query API Flows include: +
+** <<Query API>> +
+Shows how to query the Salesforce CDP data lake across data model, lake, unified, and linked objects.
+* Calculated Insights API Flows include: +
+** <<Insights - List Metadata API>> +
+Shows how to retrieve the metadata including the dimensions and measures for all calculated insights.
+** <<Insights - Get Metadata API>> +
+Shows how to retrieve the metadata including the dimensions and measures for a calculated insight.
+** <<Insights - Get Insights API>> +
+Shows how to query the calculated insights. Users can slice, dice, and filter by selecting different dimensions, measures, and filters.
+* Profile API Flows include: +
+** <<Profile - List Metadata API>> +
+Shows how to retrieve the metadata for all data model objects.
+** <<Profile - Get Metadata API>> +
+Shows how to retrieve the metadata for the requested data model object.
+** <<Profile - Search Records API>> +
+Shows how to retrieve data model object records based on search filters.
+** <<Profile - Search Records By ID API>> +
+Shows how to retrieve data model object records based on indexes and search filters.
+** <<Profile - Search Records With Child Records API>> +
+Shows how to retrieve data model object records and child object records based on indexes and search filters.
+** <<Profile - Search Records With Insight API>> +
+Shows how to retrieve data model object records and a computed view based on indexes and search filters.
+* Data Action API Flows include: +
+** <<Data Action Webhook>> +
+Shows how to run operations when a data action is triggered.
+
+
+== Insert Data - Streaming API
+
+This Mule flow shows how to insert data via the streaming API to an object's endpoint. Refer to the https://developer.salesforce.com/docs/atlas.en-us.c360a_api.meta/c360a_api/c360a_api_insert_records.htm[Salesforce API docs].
+
+This example uses the following operations:
+
+* *HTTP Listener* +
+Accepts data from HTTP POST requests with the source and object name as the URI parameters.
+* *Logger* +
+Shows the HTTP request received from the *Listener*, then later shows the HTTP response from the *Insert Objects* operation.
+* *Streaming Insert*
++
+** Authenticates based on your choice of either JWT or username and password.
+** Receives the source name, object name, and payload.
+** Calls the Salesforce API to insert the payload to that object's endpoint.
```

Comment:
This wording isn't quite right, but it's up to you whether you want to change it here and elsewhere. The meaning is clear.

Suggestion: Requests that the Salesforce API insert...